distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -1,4 +1,32 @@
-#Requires -Version 5.0
+﻿#Requires -Version 4.0
+function DownloadFiles {
```

Comment:
I got the message that the Download completed *after* the script finished. The function then hangs indefinitely with the progress bar on display.

Reply:
Seeing this now... I'll look into this. This is NOT how it was working before.

Reply:
Completely reworked this